### DISCUSSION TITLE
win 10 iis - no joy

### DISCUSSION THREAD

**Discussion by BLooRT** [ImageMagick/ImageMagick#6606] [Help]
### ImageMagick version

7.1.0.62

### Operating system, version and so on

win 10 x64 iis php 8.2.9 nts

### Description
trying to use it with wordpress and iis on localhost
everything seems to work correctly except for within iis
dos magic -version or whatever command works
i've done the things, php-imagick.ll is in c:\php\ext
and all the other dll are in c:\php
php.ini has the added entry
and the image magic installation is installed
machine is rebooted.
imagick.dll is enabled in iss php control panel

imagick section does not show up in phpinfo
c:\php doesn't show up in the in the path section of phpinfo (should it?)

i'm at a loss.  i've done the things.  no joy.  any ideas?

**Comment by snibgo** [comment]
This is an ImageMagick forum.

Your question seems to be about IMagick, which is developed by and supported by different people.

IMagick is an interface between PHP and ImageMagick.

**Reply by BLooRT** [reply]
thank you.  i sort of understand.
